llama_stack/cli/stack/run.py

```diff
@@ -168,21 +167,17 @@ def get_conda_prefix(env_name):
                     color="red",
                 )
                 return
-
-            script = importlib.resources.files("llama_stack") / "distribution/start_conda_env.sh"
-            run_args = [
-                script,
-                image_name,
-            ]
         else:
             # else must be venv since that is the only valid option left.
             current_venv = os.environ.get("VIRTUAL_ENV")
-            venv = args.image_name or current_venv
-            script = importlib.resources.files("llama_stack") / "distribution/start_venv.sh"
-            run_args = [
-                script,
-                venv,
-            ]
+            env_name = args.image_name or current_venv
```

Comment:
Good opportunity to add a print statement that we use a venv?

Reply:
good point, thanks!